src/main/java/browserview/BrowserComponent.java

```diff
@@ -123,12 +130,25 @@ private ChromeDriverEx createChromeDriver() {
         ChromeDriverEx driver = new ChromeDriverEx(options, isTestChromeDriver);
         WebDriver.Options manage = driver.manage();
         if (!isTestChromeDriver) {
+            setWindowSize(manage);
```

Comment:
`manage` may not be the most intuitive name. Seems like this is a configuration object of some kind?

`manage` as a method name is also rather vague.